_inc/client/plans/product-selector.jsx

```diff
@@ -141,12 +147,35 @@ class ProductSelector extends Component {
 		return false;
 	}
 
+	handleLandingPageLinkClick = () => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'landing-page-link',
+			feature: 'single-product-backup',
+			extra: this.state.selectedBackupType,
+		} );
+	};
+
+	setSelectedBackupType = selectedBackupType => {
+		this.setState( { selectedBackupType } );
+	};
+
 	renderTitleSection() {
+		const { backupInfoUrl } = this.props;
 		return (
 			<Fragment>
 				<h1 className="plans-section__header">{ __( 'Solutions' ) }</h1>
 				<h2 className="plans-section__subheader">
 					{ __( "Just looking for backups? We've got you covered." ) }
+					<br />
+					<ExternalLink
```

Comment:
The link will be displayed always, regardless of whether the user has a plan/product or not. 

To mirror the behavior in WP.com, we should display this link only when both conditions are true:

* The plans/products data has loaded
* The site doesn't have plan/product yet.

Reply:
A really good catch, @tyxla ! I've covered this case in 871f0e8.

<img width="535" alt="Screenshot 2019-12-16 at 11 53 06" src="https://user-images.githubusercontent.com/478735/70901490-2e2a5500-1ffb-11ea-89d0-85b4f49e4d37.png">